controllers/managedocs_controller_test.go

```diff
@@ -8,10 +8,10 @@ import (
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
 	openshiftv1 "github.com/openshift/api/network/v1"
-	ocsv1 "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
 	opv1a1 "github.com/operator-framework/api/pkg/operators/v1alpha1"
 	promv1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1"
 	promv1a1 "github.com/prometheus-operator/prometheus-operator/pkg/apis/monitoring/v1alpha1"
+	ocsv1 "github.com/red-hat-storage/ocs-operator/api/v1"
```

Comment:
- this can probably go into replace section in go.mod
- rationale being, it'll be easy to test changes for both upstream and downstream code if needed at times

Reply:
- incorrect, assumption from my side.